nsxt/resource_nsxt_policy_tier0_gateway_interface_test.go

```diff
@@ -193,6 +193,13 @@ func TestAccResourceNsxtPolicyTier0GatewayInterface_external(t *testing.T) {
 	ipAddress := "1.1.12.2"
 	updatedIPAddress := "1.2.12.2"
 	testResourceName := "nsxt_policy_tier0_gateway_interface.test"
+	var enablePim string
+	// enablePim is supported only with local manager
+	if testAccIsGlobalManager() {
+		enablePim = "false"
```

Comment:
did this work for you? GM seems to reject pim even if set to false, it needs to be absent from request

Reply:
You are right. It worked on my local setup but not on the CI.

Reply:
But still we need to update the code & doc as well as the test.